src/ui/services/customers/addNewCustomer.ui-service.ts

```diff
@@ -0,0 +1,29 @@
+import { expect, Page } from '@playwright/test';
+import { apiConfig } from 'config';
+import { STATUS_CODES } from 'data';
+import { generateCustomerData } from 'data/customers';
+import _ from 'lodash';
+import { ICustomer, ICustomerResponse } from 'types';
+import { AddNewCustomerPage, CustomersPage } from 'ui/pages';
+
+export class AddNewCustomerUIService {
+  private addNewCustomerPage: AddNewCustomerPage;
+  private customersPage: CustomersPage;
+  constructor(private page: Page) {
+    this.addNewCustomerPage = new AddNewCustomerPage(page);
+    this.customersPage = new CustomersPage(page);
+  }
```

Comment:
Можно добавить класс `PageHolder` от которого будут наследоваться все кому нужна `Page`

```suggestion

export class PageHolder {
  constructor(protected page: Page) {}
}
export class AddNewCustomerUIService extend  PageHolder {
    private addNewCustomerPage: AddNewCustomerPage = new AddNewCustomerPage(this.page);
    private customersPage: CustomersPage = new CustomersPage(this.page);
  }
```

Reply:
Это поправила для всех юай сервисов